test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -1194,34 +1192,6 @@ public static String randomDateFormatterPattern() {
         return randomFrom(FormatNames.values()).getName();
     }
 
-    /**
-     * Generate a random valid point constrained to geographic ranges (lat, lon ranges).
-     */
-    public static SpatialPoint randomGeoPoint() {
```

Comment:
I had intended to use these to replace a number of cases in Elasticsearch test code where we generate random GeoPoint (and cartesian Point), but was going to do that as a separate PR later. However, I can probably do that anyway, at low priority.

Reply:
I don't think we should have geo specific code in this class.